src/client/containers/NoteList.tsx

```diff
@@ -212,12 +212,17 @@ export const NoteList: React.FC = () => {
               <div className="note-list-outer">
                 <div data-testid={'note-title-' + index} className="note-title">
                   {note.favorite ? (
-                    <>
+                    <>                     
                       <div className="icon">
-                        <Star aria-hidden="true" className="note-favorite" size={12} />
+                        <Star aria-hidden="true" className="note-favorite" size={12} fill="currentColor"/>
```

Comment:
What is `currentColor`?

Reply:
property that takes the value of `color` property, [See docs](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentcolor_keyword)